### PR TITLE
Add arm64 signed official builds.

### PR DESCRIPTION
**Pull request by BillyONeal**

### CODE REVIEW

**Comment by BillyONeal** [comment]
https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_build/results?buildId=5283888&view=results

File: CMakeLists.txt [hunk 1]
```diff
@@ -137,6 +138,9 @@ target_compile_definitions(vcpkglib PUBLIC
     VCPKG_VERSION=${VCPKG_VERSION}
     VCPKG_BASE_VERSION=${VCPKG_BASE_VERSION}
 )
+if (WIN32)
+    target_compile_options(vcpkglib PRIVATE /EHsc)
+endif()
```
Comment (strega-nil-ms) [inline CMakeLists.txt:143]: Best to add `/EHsc` to `CMAKE_CXX_FLAGS`, I think, given that we're adding it to every C++ target; at some point before `add_library`, write `string(APPEND CMAKE_CXX_FLAGS " /EHsc")`

File: CMakeLists.txt [hunk 2]
```diff
@@ -20,6 +20,7 @@ option(VCPKG_BUILD_TLS12_DOWNLOADER "Enable building the tls12-downloader" OFF)
 option(VCPKG_BUILD_FUZZING "Option for enabling vcpkg-fuzz support" OFF)
 option(VCPKG_EMBED_GIT_SHA "Option for to fill in the Git SHA version; off by default to avoid privacy concerns out of official builds" OFF)
 option(VCPKG_ADD_SOURCELINK "Option for enabling SourceLink in debug information on Windows/MSVC builds" "${VCPKG_EMBED_GIT_SHA}")
+option(VCPKG_PDB_SUFFIX "Option to append to the name of the PDB for the shipping vcpkg binaries." "")
```
Comment (strega-nil-ms) [inline CMakeLists.txt:23]: This isn't an option (options are boolean), it should be a regular cache variable:

```suggestion
set(VCPKG_PDB_SUFFIX "" CACHE STRING "Append this string to the name of the PDB for shipping vcpkg binaries.")
```

**Review by strega-nil-ms — CHANGES_REQUESTED** [review]

**Review by strega-nil-ms — APPROVED** [review]

**Review by ras0219 — APPROVED** [review]
If we're going to be splitting on host architecture for Windows, does it make sense to also add x64?

**Comment by BillyONeal** [comment]
> If we're going to be splitting on host architecture for Windows, does it make sense to also add x64?

Do we have a use case? The original rationale for doing x86 only was that we don't do anything where perf matters and the x86 version runs on everything. ARM changed that; to my understanding the x86 on arm behavior is a lot less reasonable/transparent than the x86 on x64 behavior.

I suppose technically the WOW64 subsystem can be uninstalled on servers but that's also likely to break most of our catalog so I'm not sure how much worrying about that is worth?

**Pull request merged**